addons/account/migrations/9.0.1.1/post-migration.py

```diff
@@ -974,6 +974,80 @@ def fill_move_line_invoice(cr):
     )
 
 
+@openupgrade.logging()
+def fast_compute_cash_basis(env):
+    """Recompute faster these fields"""
+    # Replace https://github.com/OCA/OpenUpgrade/blob/7a49881f87c0aa6ef29c/addons/account/models/account_move.py#L44-L62
+    openupgrade.logged_query(
+        env.cr,
+        """
+            UPDATE account_move
+            SET matched_percentage = sub.matched_percentage
+            FROM (
+                SELECT
+                    aml.move_id,
+                    CASE
+                        WHEN 0.0 == ROUND(
```

Comment:
This is not Python, hehe

```suggestion
                        WHEN 0.0 = ROUND(
```